llvm/test/ThinLTO/X86/memprof-basic.ll

```diff
@@ -303,6 +303,23 @@ attributes #0 = { noinline optnone }
 ; IR: attributes #[[NOTCOLD]] = { "memprof"="notcold" }
 ; IR: attributes #[[COLD]] = { "memprof"="cold" }
 
+; IRNODIST: define {{.*}} @main
```

Comment:
For all the memprof tests, probably better to just loosen up the original matching a bit (by removing the `()` and/or adding `{{.*}}`)

Reply:
good point. I can do this.

Reply:
Thanks (I still see the test churn but assume you haven't had a chance to update those yet).

Reply:
The above 'IRNODIST' thing is already gone in the latest patch. The above code is marked as 'Outdated'.